inputs.tf

```diff
@@ -1,80 +1,132 @@
 }
 
-variable "kubernetes_config_file" {
+variable kubernetes_config_file { 
   description = "Kubernetes config file path."
   type        = string
   default     = "~/.kube/config"
 }
 
-variable "kubernetes_config_context" {
+variable kubernetes_config_context { 
   description = "Kubernetes config context."
   type        = string
   default     = ""
 }
 
-variable "nginx_http_port" {
+variable nginx_http_port { 
   description = "Port number for Nginx nodeport HTTP binding"
   type        = string
   default     = ""
-
-
-  /*
-  validation {
-    condition     = var.nginx_http_port == null || (1025 <= var.nginx_http_port && var.nginx_http_port <= 65535)
-    error_message = "nginx_http_port must be between 1025 and 65535"
-  }
-  */
 }
 
-variable "nginx_https_port" {
+variable nginx_https_port { 
   description = "Port number for Nginx nodeport HTTPS binding"
   type        = string
   default     = ""
+}
+
+variable nginx_extra_values {
+  type        = map(string)
+  description = "MAP of Helm values for the NGINX stack"
+}
+
+variable deploy_ingress_nginx { 
+  type        = bool
+  default     = false
+  description = "Deploy Ingress Nginx proxy"
+}
 
-  /*
-  validation {
-    condition     = var.nginx_https_port == null || (1025 <= var.nginx_https_port && var.nginx_https_port <= 65535)
-    error_message = "nginx_https_port must be between 1025 and 65535"
-  }
-  */
+variable deploy_hello_world { 
+  type        = bool
+  default     = false
+  description = "Deploy Hello World app"
+}
+
+variable loki_extra_values {
+  type        = map(string)
+  description = "MAP of Helm values for the Loki stack"
+}
+
+variable grafana_prometheus_extra_values {
+  type        = map(string)
+  description = "MAP of Helm values for the Grafana/Prometheus stack"
 }
 
-variable "deploy_tiling" {
-  description = "Whether or not to include the tiling module resources"
+variable grafana_additional_data_sources {
+  type        = list
+  description = "List of MAP specifying additional data sources for grafana, defaults to Loki data source"
+}
+
+variable promtail_extra_values {
+  type        = map(string)
+  description = "MAP of Helm values for the Promtail stack"
+
+}
+
+variable deploy_grafana_prometheus { 
   type        = bool
   default     = true
+  description = "Deploy Grafana and Prometheus stack"
+}
+
+variable deploy_loki { 
+  type        = bool
+  default     = true
+  description = "Deploy Loki stack"
+}
+
+variable deploy_promtail { 
+  type        = bool
+  default     = true
+  description = "Deploy Promtail stack"
+}
+
+variable deploy_argo_workflows { 
+  type        = bool
+  default     = true
+  description = "Deploy Argo Workflows"
+}
+
+variable deploy_argo_events { 
+  type        = bool
+  default     = false
+  description = "Deploy Argo Events"
+}
+
+variable namespace_annotations {
+  type        = map(string)
+  description = "MAP of custom defined namespace annotations"
+}
+
+variable "deploy_titiler" {
```

Comment:
maybe just change this to `variable deploy_titiler` to be consistent with the other variable names? (i.e. remove the quotes)

Reply:
Fixed, thank you!